### ISSUE TITLE
a portal for activating application actions

### ISSUE THREAD

**Issue by matthiasclasen** [flatpak/xdg-desktop-portal#212]
I've seen a number of applications poke holes in the dbus policy in order to activate actions of org.gnome.ControlCenter or org.gnome.SettingsDaemon. It might be a good idea to have a portal for this, similar to the open-uri portal.

**Comment by matthiasclasen** [comment]
A quick sketch: https://github.com/matthiasclasen/xdg-desktop-portal/tree/actions

**Comment by matthiasclasen** [comment]
A quick sketch: https://github.com/matthiasclasen/xdg-desktop-portal/tree/actions

**Comment by hadess** [comment]
Only Calendar and Contacts [poke out to org.gnome.ControlCenter](https://github.com/search?q=org:flathub+org.gnome.ControlCenter&type=Code), and only to link to the Online Accounts panel.

For gnome-settings-daemon, the only endpoint we should allow is org.gnome.SettingsDaemon.MediaKeys, for the media-keys handling. 

Applications shouldn't be allowed to do this `--talk-name=org.gnome.SettingsDaemon` (it's a [bug in the host version of gnome-settings-daemon](https://bugzilla.gnome.org/show_bug.cgi?id=781326) that should be fixed, and was already backported). Apart from Builder, I couldn't find any app that needs access to `org.gnome.SettingsDaemon`.

To sum up, I think that we should only have a special case for online accounts, or for opening specific preferences, which different portal versions could use. (though Calendar and Contacts won't work with anything but g-o-a's implementation of online accounts...)

**Comment by matthiasclasen** [comment]
i think thats the conclusion I've come to as well. This is not useful enough, and the ux is wrong
